packages/infra/client/manager/src/pull_addr_handler.rs

```diff
@@ -0,0 +1,60 @@
+use std::time::{Duration, Instant};
+
+use crate::utils::ApiResponse;
+use anyhow::*;
+use pegboard_config::{Addresses, Client};
+use tokio::sync::RwLock;
+
+/// Duration between pulling addresses again.
+const PULL_INTERVAL: Duration = Duration::from_secs(3 * 60 * 60);
```

Comment:
should this be this high? if we taint an ats node, we should have the new ats endpoints come online within a reasonable amount of time. any reason not to do it every minute?